v2/api/dataprotection/customizations/backup_vaults_backup_instance_extensions.go

```diff
@@ -80,69 +92,73 @@ func (extension *BackupVaultsBackupInstanceExtension) PostReconcileCheck(
 	}
 
 	protectionStatus := *backupInstance.Status.Properties.ProtectionStatus.Status
+	protectionStatus = strings.ToLower(protectionStatus)
 	log.V(Debug).Info(fmt.Sprintf("Protection Status is  %q", protectionStatus))
 
-	// We only want to continue if protectionStatus == ProtectionError.
-	if !strings.EqualFold(protectionStatus, protectionError) {
+	// Return success if the status is in a terminal state
+	if terminalStates.Contains(protectionStatus) {
 		log.V(Debug).Info("Returning PostReconcileCheckResultSuccess")
```

Comment:
So we continue even if `configuringprotectionfailed`?

Reply:
Yep. since its a failure, we'd let operator handle this case + set the condition and return success.

Reply:
Ok fair enough - it's basically a fatal error.